react/src/components/PostHogFeature.tsx

```diff
@@ -46,12 +46,36 @@ export function PostHogFeature({
     return <>{fallback}</>
 }
 
-function captureFeatureInteraction(flag: string, posthog: PostHog) {
-    posthog.capture('$feature_interaction', { feature_flag: flag, $set: { [`$feature_interaction/${flag}`]: true } })
+function captureFeatureInteraction({
+    flag,
+    posthog,
+    flagVariant,
+}: {
+    flag: string
+    posthog: PostHog
+    flagVariant?: string | boolean
+}) {
+    posthog.capture('$feature_interaction', {
+        feature_flag: flag,
+        feature_flag_variant: typeof flagVariant === 'string' ? flagVariant : undefined,
+        $set: { [`$feature_interaction/${flag}`]: flagVariant ?? true },
+    })
```

Comment:
We're sending `undefined` now, is this ok? If you wanted to prevent sending that altogether you could do the following

```suggestion
    const properties = { feature_flag: flag, $set: { [`$feature_interaction/${flag}`]: flagVariant ?? true } }
    if (flagVariant === 'string') properties.feature_flag_variant = flagVariant
    
    posthog.capture('$feature_interaction', properties)
```